test/functional/data/README.md

```diff
@@ -0,0 +1,42 @@
+# Various test vectors
+
+## mainnet_alt.json
+
+For easier testing the difficulty is maximally increased in the first (and only)
+retarget period, by producing blocks approximately 2 minutes apart.
+
+The alternate mainnet chain was generated as follows:
+- use faketime to set node clock to 2 minutes after genesis block
+- mine a block using a CPU miner such as https://github.com/pooler/cpuminer
+- restart node with a faketime 2 minutes later
+
+```sh
+for i in {1..2015}
+do
+ faketime "`date -d @"$(( 1231006505 + $i * 120 ))"  +'%Y-%m-%d %H:%M:%S'`" \
+ bitcoind -connect=0 -nocheckpoints -stopatheight=$i
+done
+```
+
+The CPU miner is kept running as follows:
+
+```sh
+./minerd --coinbase-addr 1NQpH6Nf8QtR2HphLRcvuVqfhXBXsiWn8r --no-stratum --algo sha256d --no-longpoll --scantime 3 --retry-pause 1
```

Comment:
@darosior depending on what operating system you're using, you may need one or more of these patches: https://github.com/pooler/cpuminer/pulls